tests/functional/clients/standalone/package_list/fixtures/alpine.py

```diff
@@ -0,0 +1,609 @@
+pkgfiles_all = {
+    "/bin": "APKFILE",
```

Comment:
we usually have a subset of packages, not all of them. Is there a particular reason this fixture has more than 10?

Reply:
I can reduce this down to a subset 👍